pkg/rid/cockroach/store.go

```diff
@@ -161,44 +162,35 @@ func (s *Store) Bootstrap(ctx context.Context) error {
 	return err
 }
 
-// GetVersion returns the current semver of the schema
+// GetVersion returns the Version string for the Database.
+// If the DB was is not bootstrapped using the schema manager we throw and error
 func (s *Store) GetVersion(ctx context.Context) (string, error) {
```

Comment:
Can we use the same function from both db-manager and inside the gRPC backend?

Reply:
I'm going to have to refactor the db-manager, I can do this in the next PR